index.html

```diff
@@ -193,11 +193,15 @@ <h3>Defining language values</h3>
  
     <p class="advisement" id="lang_values_valid"><a class="self" href="#lang_values_valid">&#x200B;</a>Be specific about what level of conformance you expect for language tags. The word "valid" has special meaning in BCP 47. Generally "well-formed" is a better choice.</p>
  
- <p>A well-formed BCP 47 language tag has hyphen-separated subtags with specific lengths and in a particular order. Valid BCP 47 language tags are well-formed but also use only subtags that are listed in the IANA Subtag Registry. Note that the IANA Subtag Registry is frequently updated with new subtags.</p>
+ <p>A <strong>well-formed</strong> BCP 47 language tag has hyphen-separated subtags with specific lengths and in a particular order. A <strong>valid</strong> BCP 47 language tag is well-formed but also is ensures that only subtags that are listed in the IANA Subtag Registry are used. Note that the IANA Subtag Registry is frequently updated with new subtags.</p>
+ 
+ <p>When specifying conformance, <q>well-formed</q> is usually preferred because the validity requirements are more difficult to manage and don't add value to the specification in question; in most cases, specifications are primarily concerned with matching tags or ranges of tags rather than the meaning of each subtag.The subtag registry is fairly sizeable and is updated with new subtags and other information sporadically throughout the year. Validation of the contents of the subtags is not required for this.</p>
```

Comment:
in "is not required for this", "for this" is a bit vague.

Reply:
fixed in rewrite